sw/host/opentitanlib/src/transport/ultradebug/spi.rs

```diff
@@ -103,10 +104,13 @@ impl Target for UltradebugSpi {
         Ok(42)
     }
 
-    fn max_chunk_size(&self) -> Result<usize> {
+    fn get_max_transfer_sizes(&self) -> Result<MaxSizes> {
         // Size of the FTDI read buffer.  We can't perform a read larger than this;
         // the FTDI device simply won't read any more.
-        Ok(65536)
+        Ok(MaxSizes {
+            read: 65536,
+            write: 65536,
+        })
```

Comment:
My source code comment is misleading: I don't believe the FTDI chip has a 64K buffer, but rather 16-bits is the size of the operand to the MPSSE commands which perform the various FTDI serial protocols.

I believe the FTDI chip to be capable of issuing full 64K reads and writes, but as I commented earlier, SPI flash parts generally wont accept more than 256 bytes per programming operation.